launch/launch/actions/execute_process.py

```diff
@@ -199,7 +199,9 @@ def __init__(
             involved.
         :param: on_exit list of actions to execute upon process exit.
         """
-        super().__init__(**kwargs)
+        super().__init__()
+        self.__respawn = kwargs['respawn'] if 'respawn' in kwargs else None
+        self.__respawn_delay = kwargs['respawn_delay'] if 'respawn_delay' in kwargs else 0
```

Comment:
@iuhilnehc-ynos why not adding `respawn` and `respawn_delay` explictly, with type annotations and documentation?

Reply:
@hidmic 
Sorry for the late response.
Maybe I just want to avoid adding an argument to ExecuteProcess constructor and minimize the updating source code.